requirements-dev.txt

```diff
@@ -8,9 +8,8 @@ asv
 cython>=0.29.21
 black==20.8b1
 cpplint
-flake8<3.8.0
+flake8
```

Comment:
@erfannariman I only modified `environment.yml` and then the hook you added modified this file for me :clap:

Reply:
That's pretty cool! Saves quite some CI errors for other devs in the future.